Sources/NetworkViewer/Common/Services/ShareService/TextShareProvider.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  TextShareProvider.swift
+//  NetworkViewer
+//
+//  Created by Mакс T on 19.11.2024.
+//
+
+import Foundation
+import UIKit
+
+final class TextShareProvider: ShareProvider {
+    var displayName: String = "text"
+    var icon: UIImage? = UIImage(systemName: "doc.on.doc")
+
+    func shareData(for operations: [NetworkViewer.Operation]) -> ShareService.Result? {
+        let jsons = operations.map { OperationMapper.jsonFrom($0) }.joined(separator: "\n\n")
+        return .text(jsons)
```

Comment:
I think we can combine json objects into a new one. So it can looks like:
```
{
  "operations": [
    { ... },
    { ... }
  ]
}
```
So we will be able to paste this text to any json formatter and view formatted json text